src/__tests__/core/web_api/helper.test.js

```diff
@@ -43,4 +34,17 @@ describe('normalizeError', () => {
     const actualError = normalizeError(error);
     expect(actualError).toMatchSnapshot();
   });
+  it('should map access_denied error to invalid_user_password when error.code === access_denied', () => {
+    const error = {
+      code: 'access_denied',
+      description: 'foobar'
+    };
+    const expectedError = {
+      code: 'invalid_user_password',
+      error: 'invalid_user_password',
+      description: error.description
```

Comment:
use `description: 'foobar'` instead of error.description

Reply:
fixed